contractcourt/channel_arbitrator.go

```diff
@@ -990,11 +990,18 @@ func (c *ChannelArbitrator) stateStep(
 		label := labels.MakeLabel(
 			labels.LabelTypeChannelClose, &c.cfg.ShortChanID,
 		)
-
 		if err := c.cfg.PublishTx(closeTx, label); err != nil {
 			log.Errorf("ChannelArbitrator(%v): unable to broadcast "+
 				"close tx: %v", c.cfg.ChanPoint, err)
-			if err != lnwallet.ErrDoubleSpend {
+			// This makes sure we don't fail at startup if the
+			// commitment transaction has too low fees to make it
+			// into mempool. The rebroadcaster makes sure this
+			// transaction is republished regularly until confirmed
+			// or replaced.
+			var mempoolFeeErr lnwallet.ErrMempoolFee
+			if !errors.Is(err, lnwallet.ErrDoubleSpend) &&
```

Comment:
hmmm I think we could get rid of the mixing of `Is` and `As` if we define `lnwallet.ErrMempoolFee` the same way as `lnwallet.ErrDoubleSpend`.

Reply:
https://github.com/lightningnetwork/lnd/pull/7746#discussion_r1245340562

So want I wanted to achieve is, to include the error message of the original error wdyt?

Reply:
ok everything is using `Is` now.